schema/me/conversation/send_message_mutation.js

```diff
@@ -0,0 +1,82 @@
+import impulse from "lib/loaders/impulse"
+import gravity from "lib/loaders/gravity"
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId, cursorForObjectInConnection } from "graphql-relay"
+import { ConversationType, MessageEdge } from "./index"
+
+const { IMPULSE_APPLICATION_ID } = process.env
+
+const getImpulseToken = gravityToken => {
+  return gravity.with(gravityToken, { method: "POST" })("me/token", {
+    client_application_id: IMPULSE_APPLICATION_ID,
+  })
+}
+
+export default mutationWithClientMutationId({
+  name: "SendConversationMessageMutation",
+  description: "Appending a message to a conversation thread",
+  inputFields: {
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The id of the conversation to be updated",
+    },
+    from: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The email address of the message sender",
+    },
+    body_text: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    reply_to_message_id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The message being replied to",
+    },
+  },
+  outputFields: {
+    conversation: {
```

Comment:
I had tried it without, but it was in fact required and future users of this mutation would probably also expect it, as the conversation node in the graph is definitely updated after adding a message.